mongo-utils/src/main/java/org/wso2/mongo/actions/Find.java

```diff
@@ -29,9 +30,21 @@
 public class Find extends AbstractMongoDBAction {
```

Comment:
Shall we bring similar java methods to a single class, since we no longer need separate classes for separate native methods?

Reply:
This effort was simply for ballerina version migration. Let's architecture the code along with the API review currently underway